docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+
+services:
+   db:
+     image: sameersbn/mysql
+     volumes:
+       - db_data:/var/lib/mysql
+     restart: always
+     environment:
+       DB_NAME: wordpress
+       DB_USER: wordpress
+       DB_PASS: wordpress
+
+   wordpress:
+     depends_on:
+       - db
+     image: wordpress:latest
+     ports:
+       - "8000:80"
+     restart: always
+     environment:
+       WORDPRESS_DB_HOST: db:3306
```

Comment:
Small detail: I believe 3306 is the port by default. No big deal, but I don't think you have to specify it